upkeep.py

```diff
@@ -174,7 +174,16 @@ def get_upcoming_workdays(workdays_ahead: int = 2) -> typing.Iterator[datetime.d
     # to help ensure the most recent existing e-scrum issue is included even when other
     # non e-scrum issues exist
     issues = repo.get_issues(state='all', sort='number', direction='desc')
-    issues = issues[:min(10 + args.workdays_ahead, issues.totalCount)]
+    # If totalCount is working properly
+    if issues.totalCount > 0:
+        issues = issues[:min(10 + args.workdays_ahead, issues.totalCount)]
+    else:
+        # Try reducing the number of issues
+        try: 
+            issues = issues[:(10 + args.workdays_ahead)]
+        # But if that's not possible, just continue
+        except:
+            pass
```

Comment:
Do you know more about the error that could occur (i.e. except a specific error) and why it happens?